src/Middleware/LoginSessionMiddleware.php

```diff
@@ -35,6 +37,13 @@ public function process(HTTPRequest $request, callable $delegate)
             // If the session has already been revoked, or we've got a mismatched
             // member / session, log the user out (this also revokes the session)
             if (!$loginSession || (int)$loginSession->MemberID !== (int)$member->ID) {
+                if (method_exists(RememberLoginHash::class, 'setLogoutAcrossDevices')) {
+                    RememberLoginHash::setLogoutAcrossDevices(false);
+                } else {
+                    // TODO: remove directly modifying config once framework 4.8 has been released
+                    // and update composer.json requirement of framework to ^4.8
+                    Config::modify()->set(RememberLoginHash::class, 'logout_across_devices', false);
+                }
```

Comment:
Target ^4.8 now and remove this. `4.x-dev` meets the `^4.8` constraint.

Reply:
Cool, TIL 4.x-dev will match ^4.8 with `"minimum-stability": "dev"` enabled .. I thought it would only match to 4.8.x-dev

This seems reasonable to remove then.  I think it does mean we cannot install using recipe 4.7.x-dev, instead we need 4.x-dev, though that's fine.

Reply:
Have updated

Reply:
FYI composer internally aliases `4.x-dev` to `4.9999999999`. That's why `4.x-dev` meets the `^4.8` constraint.